### DISCUSSION TITLE
How to get layout name in Nuxt 3 composition api <script setup>?

### DISCUSSION THREAD

**Discussion by renzivandev** [nuxt/nuxt#20281] [Questions] [answered]
In the docs, it only has setting layout, but nothing about getting the layout.

The code would look like this, but how do I get the value of layout

```
<div :class="`${layout === 'fluid' ? 'container-fluid' : 'container'}`">
</div>
```

**Comment by ndragun92** [comment, marked as answer]
I have created an example for you of how you can do that:

https://stackblitz.com/edit/github-kudwh4-bsmkaf?file=layouts%2Fdefault.vue,pages%2Faccount%2Findex.vue,layouts%2Faccount.vue,app.vue

`{{ $route.meta.layout || 'default' }}`

Hope it helps

**Reply by renzivandev** [reply]
Thanks!

**Reply by ndragun92** [reply]
const route = useRoute()

console.log('layout', route.meta.layout || 'default )
